.github/workflows/release.yml

```diff
@@ -212,3 +212,54 @@ jobs:
           body: |
             This is a "canary" release of the most recent commits on our main branch. Canary is **not stable**.
             It is only intended for developers wishing to try out the latest features in Spin, some of which may not be fully implemented.
+
+  create-go-sdk-tag:
+    name: create tag sdk/go/v*
+    runs-on: ubuntu-latest
+    needs: build
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set the tag sdk/go/v*
+        shell: bash
+        run: echo "GO_SDK_TAG=sdk/go/${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - name: Tag sdk/go/v* and push it
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git tag ${{ env.GO_SDK_TAG }}
+          git push origin ${{ env.GO_SDK_TAG }} -f
+
+  commit-and-create-template-tag:
+    name: Change sdk version in templates and create a new tag spin/templates/v*
+    runs-on: ubuntu-latest
+    needs: create-go-sdk-tag
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set the tag sdk/go/v*
```

Comment:
```suggestion
      - name: set the tag spin/templates/v*
```

Reply:
The only thing I'm unsure about is if we're okay pushing the `spin/templates/v*` tag for all `v*` releases, since the process mentions: 

```
Note the `v0.3` - the tag only needs to be created on minor releases and
   this step should be skipped on patch releases.
```

Let's double-check with @itowlson.  Also, it would be good to double-check that all of the details of this process are still needed for template versioning -- not sure if things have changed.

Reply:
Fixed the error name.